src/main/java/net/liquidpineapple/pang/logger/Logger.java

```diff
@@ -0,0 +1,77 @@
+package net.liquidpineapple.pang.logger;
+
+/**
+ * The logger class containing static methods to log out to the console.
+ * @author Jurriaan Den Toonder
+ * @date 2016/09/21.
+ */
+public class Logger {
```

Comment:
General suggestion, wouldn't it be nice to add the current timestamp to the log as this would create a more detailed overview of the time between the actions that are printed. You'd be able to see the time it takes between certain events which could be usefull during performance analysis